client/my-sites/plan-features/header.jsx

```diff
@@ -123,29 +123,29 @@ export class PlanFeaturesHeader extends Component {
 			popular,
 			selectedPlan,
 			title,
-			isInSignup,
 			audience,
 			translate,
 		} = this.props;
 
 		const headerClasses = classNames( 'plan-features__header', getPlanClass( planType ) );
+		const isPillInCorner = this.resolveIsPillInCorner();
 
 		return (
 			<span>
 				<header className={ headerClasses }>
 					<h4 className="plan-features__header-title">{ title }</h4>
 					<div className="plan-features__audience">{ audience }</div>
 					{ planLevelsMatch( selectedPlan, planType ) && (
-						<PlanPill isInSignup={ isInSignup }>{ translate( 'Suggested' ) }</PlanPill>
+						<PlanPill isInSignup={ isPillInCorner }>{ translate( 'Suggested' ) }</PlanPill>
```

Comment:
@niranjan-uma-shankar This is the only thing that stands out to me as a little odd, since the bug we had in this file before essentially resulted from properties being passed in that didn't match what you'd expect them to be.

But as far as I can see, this is only used to set a CSS class, so it's low-impact, and I trust that you've tested it does the right thing.  It sort of suggests that the property and the CSS class in the `PlanPill` component may be a bit misnamed, though (if it's not specifically about being in signup).

Reply:
Thanks for taking a look @DavidRothstein ! cc @jdc91 if this is something you'd like to tighten up if the A/B test comes out a winner and we decide to launch to all users.